index.html

```diff
@@ -41,12 +41,12 @@
                 <li> <a href="https://twitter.com/umair_ra0" target="_blank"><img src="images/twitter.svg" alt="Twitter logo" /></a></li>
                 <li><a href="https://www.linkedin.com/in/engr-umair-muqeem/" target="_blank"><img src="images/Linkedin icon.svg" alt="Linkedin logo" /></a></li>
                 <li><a href=""><img src="images/medium.svg" alt="Mediumlogo" /></a></li>
-                <li><a href="https://github.com/umair-rao" target="_blank"><img src="images/github.svg" alt="github logo" /></a></li>
-                <li><a href=""><img src="images/angellist.svg" alt="angellist logo" /></a></li>
+                <li><a href="https://github.com/umair-rao" target="_blank"><img src="images/github.svg" alt="Github logo" /></a></li>
+                <li><a href=""><img src="images/angellist.svg" alt="Angellist logo" /></a></li>
             </ul>
             </div>
             <div id="action">
-                <img src="images/Action.svg" alt="Action-icon">
+                <img src="images/Action.svg" alt="">
```

Comment:
- [ ] Kindly add alt text for this `<img>` tag for the accessibility purpose.

Reply:
Alt text added as suggested.